src/commands/infra/scaffold.ts

```diff
@@ -135,7 +135,7 @@ export const renameTfvars = async (dir: string): Promise<void> => {
   try {
     const tfFiles = fs.readdirSync(dir);
     tfFiles.forEach(file => {
-      if (file.substr(file.lastIndexOf(".") + 1) === "tfvars") {
+      if (file === "terraform.tfvars") {
```

Comment:
I would still prefer that we not copy the "tfvars" file in the first place.  If there are multiple tfvars files, the above could cause issues.

Reply:
resolved!